OPERATOR.rst

```diff
@@ -93,14 +94,36 @@ Triaging ``sandbox`` failures
   discretion on whether or not to open a new ticket, it is important to record
   unusual failures to determine their eventual frequency.
 
+Triaging GitLab build failures on ``dev`` and ``prod``
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+If a GitLab build fails on a stable deployment, the operator must evaluate the
+impact of that failure. This evaluation should include visiting the Data Browser
+to verify it isn't broken.
+
+To restore the deployment to a known working state, the operator should rerun
+the deploy job of previous passing pipeline for that deployment. This can be
+done without pushing anything and only takes a couple of minutes. The branch
+for that deployment must then be reverted to the previously passing commit.
+
+
 .. _committing the changes separately: https://github.com/DataBiosphere/azul/issues/2899#issuecomment-804508017
 
 Reindexing
 ^^^^^^^^^^
 
 The operator must check the status of the queues after every reindex for
 failures. Use ``python scripts/manage_queues.py`` to identify any failed
-messages. If failed messages are found
+messages. If failed messages are found, use ``python scripts/manage_queues.py``
```

Comment:
Should discriminate between notifcations and tallies. I think the best sequence would be notifications until the set of failed messages stabilizes, then repeat with tallies.